rebasehelper/specfile.py

```diff
@@ -455,7 +460,8 @@ def is_empty(line):
             if patch_removed:
                 # remove the line of the patch that was removed
                 self.removed_patches.append(patch_name)
-                removed_patches.append(tag.index)
+                if tag.index:
```

Comment:
Are these added `if`s necessary? As far as I am aware, only patches should get up to this point and they should always have an index, shouldn't they?

Reply:
Of course, but the type is still `Optional[int]`.